src/style.css

```diff
@@ -0,0 +1,74 @@
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir.ttc");
+}
+@font-face {
+  font-family: Avenir-Book;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
```

Comment:
Use  font-weight for `font-face` and then remove `font-family` properties for input and just inherit it